src/media/css/reviews.styl

```diff
@@ -302,6 +302,9 @@ mkt-prompt[data-modal="flag-review"] {
             left: 20px;
         }
     }
+    section.full > div.detail-flex-wrap{
```

Comment:
can you add a space before the brace? good rule of thumb is to make your code look like the surrounding code.